app/Domain/Goalcanvas/Controllers/BigRock.php

```diff
@@ -66,7 +66,6 @@ public function get($params): Response
                 $bigrock = array("id"=>'', "title" => "", "prpojectId" => "", "author" => '');
             }
 
-
```

Comment:
Few items missing from this file:
- No repo calls (they should all be service calls. If there is no service, just create a wrapper service method
- CANVAS_NAME is still being used